code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -197,7 +195,7 @@
 	var/distance = get_dist(usr,src)
 	if(istype(usr, /mob/dead/observer) || usr.stat == 2) // ghosts can see anything
 		distance = 1
-	if (src.stat)
+	if (src.stat && !(src.species.flags & NO_BLOOD))	// No point checking pulse of a species that doesn't have one.

```

Comment:
This will not only silently fail, but give this message if you try to get the heartbeat of anyone with NO_BLOOD: `span("warning", "[T.He] [T.is] not responding to anything around [T.him].\n")` Mostly because they *do* have stat.

It should give a message like, "You cannot find a pulse," or something creative like that. And rework the if/elif logic here. The else if needs to go elsewhere, I imagine.